pom.xml

```diff
@@ -1,71 +1,148 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <prerequisites>
-        <maven>3.0</maven>
-    </prerequisites>
+    <parent>
+        <groupId>org.graylog.plugins</groupId>
+        <artifactId>graylog-plugin-web-parent</artifactId>
+        <version>3.1.0</version>
+        <relativePath>../graylog2-server/graylog-plugin-parent/graylog-plugin-web-parent</relativePath>
+    </parent>
 
-    <groupId>org.graylog.plugins</groupId>
     <artifactId>graylog-plugin-pagerduty</artifactId>
-    <version>1.3.1-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Wasn't sure of how the version should look like so feel free to let me know if this would be considered a patch or major version.

Reply:
Since this is both a new type on the Graylog side as well as the PagerDuty API, I think this should to 2.0 here.